http-add-on/values.yaml

```diff
@@ -13,6 +13,37 @@ operator:
       cpu: 250m
       memory: 20Mi
   port: 8443
+  adminService: operator-admin
+  adminPort: 9090
+  routingTableURLPath: routing
+
+scaler:
+  service: external-scaler
+  pullPolicy: Always
+  grpcPort: 9090
+  healthPort: 9091
+  pendingRequestsInterceptor: 200
+
+interceptor:
+  pullPolicy: Always
+  adminService: interceptor-admin
+  adminPort: 9090
+  proxyService: interceptor-proxy
+  proxyPort: 8080
+  routingTableUpdateDurationMS: 500
+  tcpConnectTimeout: 500ms
+  keepAlive: 1s
+  responseHeaderTimeout: 500ms
+  replicasWaitTimeout: 1500ms
+  minReplicas: 1
+  maxReplicas: 50
+  pollingInterval: 1
+  deploymentCachePollingIntervalMS: 1000
+  forceHTTP2: false
+  maxIdleConns: 100
+  idleConnTimeout: 90s
+  tlsHandshakeTimeout: 10s
+  expectContinueTimeout: 1s
```

Comment:
What about this?
```suggestion
interceptor:
  pullPolicy: Always
  admin:
    service: interceptor-admin
    port: 9090
  proxy
    service: interceptor-proxy
    port: 8080
  routingTableUpdateDurationMS: 500
  tcpConnectionTimeout: 500ms
  keepAlive: 1s
  responseHeaderTimeout: 500ms
  replicasWaitTimeout: 1500ms
  replicas:
    min: 1
    max: 50
  pollingInterval: 1
  deploymentCachePollingIntervalMS: 1000
  forceHTTP2: false
  maxIdleConns: 100
  idleConnectionTimeout: 90s
  tlsHandshakeTimeout: 10s
  expectContinueTimeout: 1s
```

Reply:
done in https://github.com/kedacore/charts/pull/169/commits/9a86221e61f5f1ba36f8b1c5b47394288dc5134c